lib/presentation/screens/home/profile/profile_screen.dart

```diff
@@ -13,9 +16,212 @@ class _ProfileScreenState extends State<ProfileScreen> {
   @override
   Widget build(BuildContext context) => Scaffold(
         appBar: AppBar(
-          leading: Assets.images.icons.tinderWhite.svg(color: Colors.pink[600]),
+          leading: Assets.images.icons.tinderWhite
+              .svg(color: ColorPalette.colorPrimary100),
         ),
-        body: Center(child: Text(context.localizations.profileText)),
-        backgroundColor: Colors.teal,
+        body: getBody(context),
+        backgroundColor: ColorPalette.gray.withOpacity(0.2),
       );
+
+  Widget getBody(BuildContext context) {
+    var size = MediaQuery.of(context).size;
+    String settingText = context.localizations.settingsText;
+    return ClipPath(
+      clipper: OvalBottomBorderClipper(),
+      child: Container(
+        width: size.width,
+        height: size.height * 0.6,
+        decoration: BoxDecoration(color: ColorPalette.white, boxShadow: [
+          BoxShadow(
+            color: ColorPalette.gray.withOpacity(0.1),
+          ),
+        ]),
+        child: Padding(
+          padding: const EdgeInsets.only(left: 30, right: 30, bottom: 40),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Container(
+                width: 140,
+                height: 140,
+                decoration: const BoxDecoration(
+                  shape: BoxShape.circle,
+                  image: DecorationImage(
+                      image: NetworkImage(
+                          "https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/david-dobrik-attends-build-brunch-to-discuss-his-recent-and-news-photo-1616503401.?crop=1xw:0.66667xh;center,top&resize=640:*"),
+                      fit: BoxFit.cover),
+                ),
+              ),
+              const SizedBox(
+                height: 15,
+              ),
+              const Text(
+                'David, 22',
+                style: TextStyle(
+                  fontSize: 25,
+                  color: ColorPalette.black,
+                  fontWeight: FontWeight.w600,
+                ),
+              ),
+              const SizedBox(
+                height: 20,
+              ),
+              Row(
```

Comment:
Ok so for those icons:
1. Definitely too much widget nesting,
2.  Create method:` Widget _profileIconButton(Icons icon, String text, bool isBig) `

- And in this widget you will have:

```
Column(
                    children: [
                      Container(
                        width: isBig? 85: 60,
                        height: isBig?85:60,
                        decoration: BoxDecoration(
                          color: ColorPalette.white,
                          shape: BoxShape.circle,
                          boxShadow: [
                            BoxShadow(
                              color: ColorPalette.gray.withOpacity(0.2),
                              blurRadius: 15,
                              spreadRadius: 10,
                            ),
                          ],
                        ),
                        child: const Icon(
                         icon,
                          size: isBig ? 45 : 35, //IconDimension create will be nice ;)
                          color: isBig? ColorPalette.colorPrimaryBase :  ColorPalette.gray,
                        ),
                      ),
                      const SizedBox(height: PaddingDimension.small),
                      Text(
                       text,
                        style:context.theme.textSheme.bodyText1 // change style with copyWith for example or take different one
                      )
                    ],
                  ),
```
- Ofc with this + on the centered icon you can wrap this in the stack and as a second child you will have` if(isBig) plusBadgeWidget,`
- With this, you are reusing the code and it will look even cleaner in the next step
3. You should simplify with these nested columns rows, etc to just 1 stack. This stack should have his alignment property set as a center (this will handle the center icon on its own) and for those left-right icons wrap them with `Alignment()` (or Align I'm not sure which of those it was) or like you tried with `Positioned()` but use the values from `size.width*0.1` for example instead of those magic numbers. You can change all of the magic numbers to correlate with screen size but it's ok in the rest of the places. But here it should depend heavily on the device.

Reply:
And the body will look sth along those lines:
```
Stack(
alignment: Alignment.center,
children: [ 
Align/Alignment/Positioned( CODE: left, child: _profileIconButton(leftIcon, leftText, false)),
_profileIconButton(centerIcon,centerText, true),
Align/Alignment/Positioned( CODE: right, child: _profileIconButton(rightIcon, rightText, false)),
],
```

Reply:
adding crosAxisAlignment stretch to parent column will let the stack use 100% of available width

Reply:
The 'Add media' button is much more complicated than the two other buttons. Imo this wouldn't be readable if we do those three as one function whos returns widget (and btw i can't do it :'( )